merk/src/tree/link.rs

```diff
@@ -134,7 +134,15 @@ impl Link {
     /// if any (note: not the height of the referenced tree itself). Return
     /// value is `(left_child_height, right_child_height)`.
     #[inline]
-    pub fn height(&self) -> u8 {
+    pub const fn height(&self) -> u8 {
+        const fn max(a: u8, b: u8) -> u8 {
```

Comment:
is it due to lack of const version of `cmp::max` ?

Reply:
correct